src/nvim/terminal.c

```diff
@@ -783,11 +783,11 @@ static bool send_mouse_event(Terminal *term, int c)
     bool drag = false;
 
     switch (c) {
-      case K_LEFTDRAG: drag = true;
+      case K_LEFTDRAG: drag = true;  /* FALLTHROUGH */
```

Comment:
Sorry if I wasn't clear, but we only use C++ style comments (especially here, since it's only one line, and they're used everywhere else in `terminal.c`).